regolith/helpers/l_projectahelper.py

```diff
@@ -104,9 +106,16 @@ def sout(self):
         bad_stati = ["finished", "cancelled", "paused", "back_burner"]
         projecta = []
         end_projecta = []
+        grp_by_lead = {}
         if rc.lead and rc.person:
             raise RuntimeError(f"please specify either lead or person, not both")
         for projectum in self.gtx["projecta"]:
+            if rc.grp_by_lead:
+                if projectum.get('lead') not in grp_by_lead:
```

Comment:
it is a little confusing to have ``grp_by_lead`` as a bool and ``rc.grp_by_lead`` as a dict.  Can you come up with better naming that makes the code more readable?